src/components/Observations/zone/index.tsx

```diff
@@ -40,11 +61,18 @@ export default function ObservationPlantingZone(): JSX.Element {
         observationId: Number(observationId),
         plantingZoneId: Number(plantingZoneId),
         search,
+        plotType: plotTypeSearch,
       },
       defaultTimeZone.get().id
     )
   );
 
+  useEffect(() => {
+    if (filters.plotType && plotTypeSearch !== undefined) {
```

Comment:
the values here are string values with semantics like permanent/temporary and need to be updated when the local changes.. so the pills reflect translated strings